src/NuGet.Core/NuGet.Common/ExceptionUtilities.cs

```diff
@@ -14,20 +14,42 @@ namespace NuGet.Common
     /// </summary>
     public static class ExceptionUtilities
     {
-        public static void HandleException(Exception ex, ILogger logger)
+        /// <summary>
+        /// Log an exception to an ILogger.
+        /// This will log using NU1000 if the exception does not contain a code.
```

Comment:
No, I'll remove this and add an undefined code so top level these can still be printed without a code.